jfrog_platform_cloud/README.md

```diff
@@ -20,6 +20,88 @@ These logs will allow customers to readily know who accessed what repositories a
 
 JFrog's SaaS Log Streaming is currently in beta. While in beta, the cloud log streaming feature will only be available inside the MyJFrog portal for select JFrog Enterprise and customers. JFrog plans to GA this feature later in Q2 2024 at which point it will be available to all JFrog Enterprise and customers.
 
+## Widgets description
```

Comment:
Can you provide more insight on the changes? Is this referring to dashboard widgets? If so, I recommend including notes beside the graphs like our [best practices guide](https://datadoghq.dev/integrations-core/guidelines/dashboards/#best-practices) recommends instead of the readme sections given the length here.

Reply:
Yes, it's referring to widgets. That's a good idea, I will add another widget with the descriptions, thanks!

Reply:
Thanks for clarifying! You're welcome. You can actually add a Notes Widget next to the graphs. The color of the notes should be the light version of the group color. ex: Group is Green, notes within the group should be light green!  

Since you are adding notes, let's remove the widget explanations here

Reply:
It's done, please check.